### PR TITLE
[BOP-1122]inline text area

### PR DESCRIPTION
**Pull request by idanzonshein**
#### Screens
Screen Shot 2021-09-07 at 9 27 21 AM
Screen Shot 2021-09-07 at 9 27 48 AM

#### Breaking Changes

[Yes/No (Explain)]

#### Runway Ticket URL

[Runway-ticket](https://nitro.powerhrg.com/runway/backlog_items/BOP-1122)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

**Comment by idanzonshein** [comment]
**Question**
This prop is being added to the kit so that long runway backlog item titles will be able to wrap. Only issue now is that we need to have this blur with the click of the enter key. Does it makes sense to have that functionality added with a prop or should that be handled with custom code on the runway side of things?
@jasperfurniss

File: playbook/app/pb_kits/playbook/pb_textarea/_textarea.jsx [hunk 1]
```diff
@@ -58,8 +60,9 @@ const Textarea = ({
   })
 
   const errorClass = error ? 'error' : null
+  const InlineClass = inline ? 'inline' : ''
```
Comment (haydenrou) [inline playbook/app/pb_kits/playbook/pb_textarea/_textarea.jsx:63]:
```suggestion
  const inlineClass = inline ? 'inline' : ''
```

File: playbook/app/pb_kits/playbook/pb_textarea/_textarea.scss [hunk 2]
```diff
@@ -67,6 +67,15 @@
     }
   }
 
+  &.inline {
+    textarea {
+      border-color: transparent;
+    }
+    textarea:focus {
+      border-color: $primary
```
Comment (haydenrou) [inline playbook/app/pb_kits/playbook/pb_textarea/_textarea.scss:75]:
```suggestion
      border-color: $primary;
```

**Review by haydenrou — APPROVED** [review]

**Comment by idanzonshein** [comment]
> **Question**
> This prop is being added to the kit so that long runway backlog item titles will be able to wrap. Only issue now is that we need to have this blur with the click of the enter key. Does it makes sense to have that functionality added with a prop or should that be handled with custom code on the runway side of things?
> @jasperfurniss

This was discussed and answered when meeting with jasper. It will be taken care of on the runway side

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**